build.cake

```diff
@@ -558,7 +558,10 @@ void RunNUnitTests(DirectoryPath workingDir, string testAssembly, string framewo
     try
     {
         var path = workingDir.CombineWithFilePath(new FilePath(testAssembly));
-        NUnit3(path.ToString());
+        var settings = new NUnit3Settings();
+        if(!IsRunningOnWindows())
```

Comment:
This bothers me a little. Is it purely a limitation of the NUnit3 extension? Or does it matter what version of the runner we use with the tool?